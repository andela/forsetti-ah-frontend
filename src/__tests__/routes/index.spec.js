import React from 'react';
import { shallow } from 'enzyme';
import Routes from '../../routes';

describe('Routes', () => {
  test('should test default route', () => {
    const wrapper = shallow(<Routes />);
    expect(wrapper).toMatchSnapshot();
  });
});