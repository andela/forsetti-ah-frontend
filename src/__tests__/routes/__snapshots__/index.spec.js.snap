// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes  should test article route 1`] = `ShallowWrapper {}`;
